java/java-quiz.md

```diff
@@ -1881,3 +1881,17 @@ numbers                         // Input `List<Integer>` > [1, 2, 3, 4] <br>
  - [ ] out
  - [ ] err
  - [ ] in
+
+#### Q127. The compiler is complaining about this assignment of the variable pickle to the variable jar. How woulld you fix this?
+
+- [ ] Use the method toInt() to convert pickle before assigning it to jar.
+- [x] Cast pickle to an int before assigning it to jar.
+- [ ] Make pickle into a double by adding + ".0"
+- [ ] Use the new keyword to create a new Integer from pickle before assigning it to jar.
+
+#### Q128. What value should x have to make this loop execute 10 times?
+
```

Comment:
please add here the code from the question 
![image](https://user-images.githubusercontent.com/13170022/172116737-3cce796f-03cb-4648-bea6-1b91781d2e56.png)